app/Queries/TradingUsersQuery.php

```diff
@@ -18,7 +18,11 @@ public function fetchAll(User $user, int $limit = 5)
             return $elephpant->pivot->quantity > 1;
         });
 
-        $traders = $this->fetchTraders($userElephpants, $limit);
+        if ($userAvailable->count() == 0) {
+            return [];
```

Comment:
it would be good to have the result sent to the view, then we can add a custom message to the user like "You don't have any double elePHPant to trade yet" or something like this.